eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -318,6 +323,7 @@ private boolean checkIfExistingRSOrCandidateForMerge(SubmittedVariantEntity subm
                 submittedVariantOperationEntity.fill(EventType.RS_MERGE_CANDIDATES, accessionInDB,
                         "RS mismatch with " + accessionInDB, inactiveObjects);
 
+                updateMergeCandidateSVOE.put(variantHash, submittedVariantOperationEntity);
```

Comment:
This map should be populated [here](https://github.com/EBIvariation/eva-accession/pull/315/files#diff-b6244e9907d2b0296d3a02a3fc6481f3fb8656821add15fea30f85a243ce3c59R314) too..